client/app/views/contacts/merge.coffee

```diff
@@ -11,8 +11,7 @@ module.exports = class MergeView extends Mn.ItemView
     attributes:
         role: 'alertdialog'
 
-    behaviors: ->
-        Navigator: {}
+    behaviors:
```

Comment:
I don't think so: Marionette does a `_.result` on `this` instance to get a new config object: http://devdocs.io/marionette/marionette.behavior (plus, behaviors are just blueprints, I presume it's not a problem if a same config object is shared between instances :wink:)